WcaOnRails/app/models/statistics.rb

```diff
@@ -0,0 +1,167 @@
+module Statistics
+  TextTd = Struct.new(:text) do
+    def render
+      "<td>#{text}</td>".html_safe
```

Comment:
So why manually generate HTML in a model?

Shouldn't this be view helper code?